### ISSUE TITLE
Wish for Cupertino-based samples featured on https://flutter.github.io/samples/#

### ISSUE THREAD

**Issue by sethladd** [flutter/samples#2408]
### Use case

Teams, businesses, and developers will want to see that Cupertino-flavored apps are on-par with Material-flavored apps, and will want to see Cupertino-flavored apps as the context for other types of interesting functionality.

### Proposal

Over in https://flutter.github.io/samples/#, which you can get to from our marketing pages such as https://flutter.dev/multi-platform/mobile and https://docs.flutter.dev/, should feature Cupertino-flavored apps more prominently. For example, for the Form App, it would be good to see a Cupertino-based form app as well.

**Comment by ericwindmill** [comment]
The samples repo is my responsibility @sethladd. I'll look into this

**Comment by sethladd** [comment]
Thanks Eric! 

(and thanks @darshankawar for moving this issue over, appreciate the help!)

**Comment by sethladd** [comment]
To make sure the next step for this is clear, I think the next question is "Is there today a sample that shows off our Cupertino library and thus looks iOS-y?" 

If so, this issue becomes "update https://flutter.github.io/samples with a call out to that sample".

If we don't have said sample, this issue becomes "create a sample of an iOS-y app and then add it to the samples listing on flutter.github.io/samples"

WDYT?

**Comment by ericwindmill** [comment]
I think the recently added [Date Planner app](https://github.com/flutter/samples/tree/main/experimental/date_planner) is intended to be an iOS-first sample, and we can add that to the Samples gallery.

My two questions:
- @johnpryan - Do we own the github.io/samples site? I don't know if this is different then the Visual Samples Index, but it seems like there are less samples on there than there used to be. Do you know how we edit the samples that are listed?
- @Piinks or @mit-mit - Is the date_planner app ready to be featured?

Separately, FWIW, I thought we were shutting down the github.io/samples website.  Either way, I plan to replace the website with something that doesn't require maintenance in the next quarter or two when I holistically audit our samples.

**Comment by mit-mit** [comment]
I don't think it's quite ready yet; there are a bunch of known issues. The team is pushing hard on iOS fidelity, so hopefully soon.

But I'll let @sethladd make the call here.

**Comment by Piinks** [comment]
The date planner is not ready yet, which is why we put it in `experimental/` for now. :)

**Comment by sethladd** [comment]
Thanks all! I'm hearing this conclusion: "when date_planner is ready and graduates from experimental, then include it in the gallery of samples, which at the time of writing is flutter.github.io/samples"

@Piinks  Is there an Umbrella issue for date_planner? if so, I will be happy to create a child issue that encapsulates the above conclusion, and then I can close this one out.

(@ericwindmill when a new sample gallery site/page/thing launches, we can then change that child issue)

**Comment by Piinks** [comment]
> @Piinks Is there an Umbrella issue for date_planner? if so, I will be happy to create a child issue that encapsulates the above conclusion, and then I can close this one out.

No, AFAIK we're not actively working on the date_planner app right now. Rolodex is queued as the next app to work on. I don't think we are able to work on multiple apps at the same time currently.

Umbrella issues are not ideal. We've been filing individual issues for things we have identified to work on, and are tracking them in the [Cupertino refresh project](https://github.com/orgs/flutter/projects/155/views/6?groupedBy%5BcolumnId%5D=123151514). That is the source of truth, where we have been able to organize issues by the various areas of the library.

**Comment by sethladd** [comment]
ack on sequencing here, and ack on umbrella issues.

I will then rename this issue to be specific about the date_planner, just to have a more clear title.

Thank you!

**Comment by ericwindmill** [comment]
FYI - The existing Sample index is being deprecated in favor of a page on docs.flutter.dev.
